src/core/sys/posix/arpa/inet.d

```diff
@@ -158,6 +158,31 @@ else version (NetBSD)
     const(char)*    inet_ntop(int, in void*, char*, socklen_t);
     int             inet_pton(int, in char*, void*);
 }
+else version (OpenBSD)
+{
+    alias uint16_t in_port_t;
+    alias uint32_t in_addr_t;
+
+    struct in_addr
+    {
+        in_addr_t s_addr;
+    }
+
+    enum INET_ADDRSTRLEN    = 16;
+
+    @trusted pure
+    {
+    uint32_t htonl(uint32_t);
+    uint16_t htons(uint16_t);
+    uint32_t ntohl(uint32_t);
+    uint16_t ntohs(uint16_t);
```

Comment:
Should be indented.

Reply:
do you want me to fix the indentation for the code in the other versions sections as well (i.e.: lines 75-78, 100-103, 125-128, etc)?  I ask because their indentations are the same as this section.

Reply:
That should probably be fixed in a separate PR.

Reply:
fixed.